bedrock/firefox/views.py

```diff
@@ -526,3 +527,51 @@ def ios_testflight(request):
     return l10n_utils.render(request,
                              'firefox/testflight.html',
                              {'newsletter_form': newsletter_form})
+
+
+class FirefoxProductDesktopView(BlogPostsView):
+    blog_posts_limit = 3
+    blog_posts_template_variable = 'articles'
+    blog_slugs = 'firefox'
```

Comment:
Are we missing `blog_tags` here? The hub questions doc seems to suggest this page should be using `blog_tags = [‘browser’, ‘featured’]`?